assets/create-container-image-job.yaml

```diff
@@ -23,6 +23,8 @@ spec:
           value: "${NUM_USERS}"
         - name: PREFIX
           value: "${PREFIX}"
+        - name: CONCURRENT_JOBS
```

Comment:
I'm not sure if this works as expected but I think this file should just be reverted and the solution Joe recently added to do Podman pushes with data stored in elasticsearch should be used.